src/registrar/tests/test_nameserver_error.py

```diff
@@ -10,10 +10,7 @@ class TestNameserverError(TestCase):
     def test_with_no_ip(self):
         """Test NameserverError when no ip address is passed"""
         nameserver = "nameserver val"
-        expected = (
-            f"Nameserver {nameserver} needs to have an "
-            "IP address because it is a subdomain"
-        )
+        expected = "Subdomains require an IP address"
```

Comment:
Thinking this might have to change to

> Using your domain for a name server requires an IP address


Hopefully this is the right place

Reply:
done